app/components/orders/order-summary.js

```diff
@@ -14,6 +14,13 @@ export default Component.extend(FormMixin, {
     );
   }),
 
+  taxed_price: computed('data.amount', 'event.tax.rate', function() {
```

Comment:
@pradeepgangwar I've set the order amount just after the price+tax had been calculated. But the server data still shows up to be the old data.